packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div ref="range" type="range" class="sf-range" :disabled="disabled">
+    <slot v-bind="$attrs" />
+  </div>
+</template>
+<script>
+import noUiSlider from "nouislider";
+export default {
+  name: "SfRange",
+  props: {
+    /*
+     * Sets the number of sliders (1 or 2) by adding initial values to the array
+     */
+    value: {
+      type: [Number, Array],
+      default: () => [0, 1],
+    },
+    /*
+     * Disabling the slider
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    /*
+     * Settings for noUiSlider library
+     */
+    config: {
+      type: Object,
+      default: () => {
+        return {
+          start: [0, 1],
+          range: {
+            min: 0,
+            max: 10,
+          },
+          step: 1,
+          connect: true,
+          direction: "ltr",
+          orientation: "horizontal",
+          behaviour: "tap-drag",
+          tooltips: false,
+          keyboardSupport: true,
+        };
+      },
+    },
+  },
+  watch: {
+    config: {
```

Comment:
Do we expect config to change over time? I think it should be initialized once and used throughout the app. Correct me if I am wrong ;)

Reply:
For norrmal use cases you are right, but I also consider what is going on in Storybook, when data can change. If I shouldn't do this I'll remove this part.